frontend/src/index.tsx

```diff
@@ -17,15 +17,31 @@ import {ReactComponent as HomeBtn} from "./images/home.svg";
 
 function NavComponents({accountToken}: {accountToken: string | null}) {
     const {setAccountToken} = useAccountManagement();
-    const navgiate = useNavigate();
+    const navigate = useNavigate();
 
     const logout = () => {
-        setAccountToken(null);
-        if (config.rctf_mode && config.rctf_url !== null) {
-            window.location.href = config.rctf_url;
-        } else {
-            navgiate("/");
-        }
+        fetch("/api/accounts/logout", {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json",
+            },
+            body: JSON.stringify({
+                login_token: accountToken,
+            }),
```

Comment:
all api endpoints require authentication by default (unless you explicitly disable it in `__init__.py`) - make sure you add the account session token as an authentication bearer token header as well